src/execution_plan/ops/op_value_hash_join.c

```diff
@@ -133,6 +133,7 @@ void _cache_records(OpValueHashJoin *op) {
 	// As long as there's data coming in from left branch.
 	do {
 		// Add joined value to record.
+		Record_PersistScalars(r);
```

Comment:
Why was this line added?

Reply:
we have a join op calling cond_var_len_traverse. The join op collects the records from the cond_var_len_traverse op but the cond_var_len_traverse op releases its internal record once it needs to create a new one (upon each consume operation). if the join op will not persist the values we will have invalidate access errors to a deleted value.

Reply:
So no change here? we didn't managed to find a better solution?